### ISSUE TITLE
Standard music widget for Android

### ISSUE THREAD

**Issue by GuiSousa135** [Malopieds/InnerTune#179]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [x] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

InnerTune needs a standard music widget (album cover, music title, artist name, controls), with more control and scaling options, like Spotify or AntennaPod widget

### Why do you want this feature?

Music playback control can be done via the homescreen

### Additional information

![244383940-7b180a58-862f-4efb-b726-dc470f486f0a](https://github.com/Malopieds/InnerTune/assets/55188286/6e970aa7-d403-4f40-ab61-e3fe7d88524b)

**Comment by pdimu** [comment]
Why not a MD3 style? Spotify doesn't have great widgets

**Comment by GuiSousa135** [comment]
Yeah, to use Material Design would make it even more elegant and consistent with InnerTune's proposal, using the Spotify widget was just an example

**Comment by pdimu** [comment]
https://github.com/Malopieds/InnerTune/issues/253#issuecomment-2247302615
> > I don't think I will do this. That is too much of work in my opinion. If someone implement it I will gladly look at their pull request !
> 
> i'll look up on google on how to make a widget. is this style good? 
![image](https://private-user-images.githubusercontent.com/128509556/351650415-94fdd41f-8e7a-40ef-a7dc-42a484bb1991.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4MTIyMjEsIm5iZiI6MTcyMTgxMTkyMSwicGF0aCI6Ii8xMjg1MDk1NTYvMzUxNjUwNDE1LTk0ZmRkNDFmLThlN2EtNDBlZi1hN2RjLTQyYTQ4NGJiMTk5MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyNFQwOTA1MjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00ZmJiODZmODBjZjg2OTBhNWQxYzE3NmM2OWUwZWNiZWZjYTE5ZjFmYTM4NmRhMjgxZDg3ZDA0ZmNhZWRmY2M3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.MAdbjx-OY_1fzDGZaXtKmK-oVreKE54TGgSKna_jIjM) 
or do i need to make it as a miniplayer?

sorry for the confusion @Malopieds

**Comment by GuiSousa135** [comment]
I really liked the two styles presented (both horizontal and vertical) with Material Design, more options is always better

**Comment by rahulganes** [comment]
![YouTube-Music-Turntable-widget-2](https://github.com/user-attachments/assets/acb13726-26b9-466b-88f5-cc181d07b637)

This turntable widget of YT Music is good. Instead of the like option, we can have a skip forward option. Can we have in this app ?

**Comment by GuiSousa135** [comment]
> ![YouTube-Music-Turntable-widget-2](https://private-user-images.githubusercontent.com/40782985/358990821-acb13726-26b9-466b-88f5-cc181d07b637.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ3NzY5MTUsIm5iZiI6MTcyNDc3NjYxNSwicGF0aCI6Ii80MDc4Mjk4NS8zNTg5OTA4MjEtYWNiMTM3MjYtMjZiOS00NjZiLTg4ZjUtY2MxODFkMDdiNjM3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI3VDE2MzY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk3M2QwMmNjOWE0YTQ3OTRlNWZmZmIxZTg4ZWJhYTY2MmQ4MDczMjA2NmI4OWU2OGFhYTcxYzRkNWFlNTVjMjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.sG9bnpd6k_SBCcQXr5HJKuiYw6Ytyl93C9sEOjqAql0)
> 
> This turntable widget of YT Music is good. Instead of the like option, we can have a skip forward option. Can we have in this app ?

This model with MD3 is very beautiful and minimalist, something cool would be to also show the name of the album, not just the cover, it would be Artist, Album and Song, with cover and control buttons.

**Comment by GuiSousa135** [comment]
maybe like this

![Image](https://github.com/user-attachments/assets/55285f3b-31e5-4f76-9056-f20226996e94)
